src/ui/Editor.cpp

```diff
@@ -795,6 +812,81 @@ Editor::IsTextSelected()
 }
 
 
+void
+Editor::LoadEditorConfig()
+{
+	fEndOfLine = EndOfLine();
+	fIndentStyle = (bool)gCFG["tab_to_space"] ? IndentStyle::Space : IndentStyle::Tab;
+	fIndentSize = (int)gCFG["tab_width"];
+	fTrimTrailingWhitespace = (bool)gCFG["trim_trailing_whitespace"];
+	fHasEditorConfig = false;
+
+	if (!(bool)gCFG["ignore_editorconfig"]) {
+		// start parsing
+		// Ignore full path error, whose error code is EDITORCONFIG_PARSE_NOT_FULL_PATH
+		editorconfig_handle handle = editorconfig_handle_init();
+		int err_num;
+		if ((err_num = editorconfig_parse(FilePath().String(), handle)) != 0 &&
+				err_num != EDITORCONFIG_PARSE_NOT_FULL_PATH) {
+			editorconfig_handle_destroy(handle);
+			LogError("Can't load .editorconfig with error %d", editorconfig_get_error_msg(err_num));
+			fHasEditorConfig = false;
+		} else {
+
+			fHasEditorConfig = true;
+
+			int32 i;
+			int32 name_value_count;
```

Comment:
Style : nameValueCount instead of name_value_count.
Aldo err_num just some lines above

Reply:
done